src/modules/Dropdown/Dropdown.js

```diff
@@ -687,6 +693,8 @@ export default class Dropdown extends Component {
   // ----------------------------------------
 
   renderText = () => {
+    if (this.props.trigger) return this.props.trigger
```

Comment:
I was thinking about this as well, we could/should probably just put this directly in the render() so it is more clear what is happening:

```jsx
{this.trigger || this.renderText()}
```